src/components/shared/Card/CardStyled.js

```diff
@@ -36,30 +37,45 @@ const CardLeft = styled.section`
 
 const CardRigth = styled.section`
         width: 85%;
-        height: 100%;
+        min-height: 260px;
         display: flex;
         flex-direction: column;
         justify-content: space-around;
-        padding: 0 0 10px 0;
 
         & .username {
             font-size: 19px;
             cursor: pointer;
+            padding: 0 0 10px 0;
+
         }
 
-        & .description {
+        & .descriptionContainer {
             width: 90%;
-            height: 53px;
+            margin-bottom: 20px;
+        }
+
+        & .description {
+            width: 100%;
             font-size: 17px;
             color: #B7B7B7;
-            overflow: hidden;
-            text-overflow: ellipsis;
+        }
+
+        & .showText {
+            color: li
         }
 
         & .description span{
             font-weight: 700;
             color: white;
         }
+
+        & .seeMore {
+            padding: 0 10px;
+            cursor: pointer;
+        }
+        & .seeMore:hover {
+            color: lightgray
+        }
         @media (max-width: 992px) {
             width: 81.6%;
             padding: 0 0 10px 0px;
```

Comment:
É preciso limpar o min-height aqui tambem.
```suggestion
            padding: 0 0 10px 0px;
            min-height: 232px;
```